spec/models/requests/request_spec.rb

```diff
@@ -280,6 +280,24 @@
     end
   end
 
+  context "A system id that has a holding with item on reserve" do
+    let(:user) { FactoryGirl.build(:user) }
+    let(:params) {
+      {
+        system_id: '8179402',
+        user: user
+      }
+    }
+    let(:request_with_items_on_reserve) { described_class.new(params) }
+    subject { request_with_items_on_reserve }
+
+    describe "#requestable" do
```

Comment:
I think adding a test for on_reserve = false will turn the coveralls readout to green.